src/libraries/System.Net.Quic/tests/FunctionalTests/QuicStreamTests.cs

```diff
@@ -228,84 +174,60 @@ public async Task GetStreamIdWithoutStartWorks()
 
             using QuicStream clientStream = clientConnection.OpenBidirectionalStream();
             Assert.Equal(0, clientStream.StreamId);
+
+            // TODO: stream that is opened by client but left unaccepted by server may cause AccessViolationException in its Finalizer
```

Comment:
`AccessViolationException` is always wrong (memory corruption), we should not be causing that in any case.
By any chance do you have details of this exception? Dump maybe?